### DISCUSSION TITLE
How do I create a specified flow boundary

### DISCUSSION THREAD

**Discussion by Alvin-prc** [modflowpy/flopy#1772] [Q&A] [answered]
I want to create a two-dimensional confined aquifer model with impermeable boundaries at the top and bottom, a constant head boundary on the left, and a specified flow boundary on the right. In the areas where the permeability coefficient is larger at the right boundary, the pumping rate is -5m3 per day, and in the areas where the permeability coefficient is smaller, the pumping rate is -1m3 per day. Is the code I have written correct?
```python
# CHD
###################################################################################
shead_0 = 102.
ehead_0 = 102.
bound_sp_0 = []

for lay in range(nlay):
    for row in range(nrow):
        bound_sp_0.append([lay, row, 0, shead_0, shead_0])

chd_spd = {0: bound_sp_0}
chd = flopy.modflow.ModflowChd(model=mf, stress_period_data=chd_spd)
#########################################################
# 5. GHB general head boundary
#########################################################
stageright = 100.
bound_sp1 = []
for il in range(nlay):
    for ir in range(nrow):
        if hk[il, ir, ncol-1] > 1:
            condright = -5.
        else:
            condright = -1.
        bound_sp1.append([il, ir, ncol - 1, stageright, condright])  
print("Adding ", len(bound_sp1), "GHBs for stress period 1.")

stress_period_data = {0: bound_sp1}
# Create the flopy ghb object
ghb = flopy.modflow.ModflowGhb(mf, stress_period_data=stress_period_data)
```

**Comment by ccalvocm** [comment, marked as answer]
Greetings Alvin-prc:

Your syntax is correct, you are creating a CHD boundary condition on the left and a GHB on the right, whose conductance varies according to hk values. 

![alt text](https://raw.githubusercontent.com/ccalvocm/Personal/master/Groundwater/imgs/chd.png)
![alt text](https://raw.githubusercontent.com/ccalvocm/Personal/master/Groundwater/imgs/ghb.png)

However, if you have pumping wells, I would use the WEL package (https://flopy.readthedocs.io/en/3.3.5/source/flopy.modflow.mfwel.html)

**Reply by Alvin-prc** [reply]
Thank you for your answer
